lang/cs/Org.Apache.REEF.IO.Tests/TestAzureDataLakeFileSystemE2E.cs

```diff
@@ -0,0 +1,229 @@
+            {
+                _adlsClient.DeleteRecursive($"/{defaultFolderName}");
+            }
+        }
+
+        [Fact(Skip = SkipMessage)]
+        public void TestOpenE2E()
+        {
+            string fileName = UploadFromString(ContentsText);
+            var stream = _fileSystem.Open(PathToFile(fileName));
+            StreamReader reader = new StreamReader(stream);
+            string streamText = reader.ReadToEnd();
+            Assert.Equal(ContentsText, streamText);
+        }
+
+        [Fact(Skip = SkipMessage)]
+        public void TestCreateE2E()
+        {
+            string fileName = $"/{defaultFolderName}/TestCreateE2E.txt";
+            var stream = _fileSystem.Create(PathToFile(fileName));
+            Assert.True(_adlsClient.CheckExists(fileName));
+            Assert.Equal(typeof(AdlsOutputStream), stream.GetType());
+        }
+
+        [Fact(Skip = SkipMessage)]
+        public void TestDeleteE2E()
+        {
+            string fileName = UploadFromString(ContentsText);
+            Assert.True(_adlsClient.CheckExists(fileName));
+            _fileSystem.Delete(PathToFile(fileName));
+            Assert.False(_adlsClient.CheckExists(fileName));
+        }
+
+        [Fact(Skip = SkipMessage)]
+        public void TestExistsE2E()
+        {
+            string fileName = UploadFromString(ContentsText);
+            Assert.True(_fileSystem.Exists(PathToFile(fileName)));
+            _adlsClient.Delete(fileName);
+            Assert.False(_fileSystem.Exists(PathToFile(fileName)));
+        }
+
+        [Fact(Skip = SkipMessage)]
+        public void TestCopyE2E()
+        {
+            throw new NotImplementedException();
+        }
+
+        [Fact(Skip = SkipMessage)]
+        public void TestCopyToLocalE2E()
+        {         
+            var tempFilePath = Path.GetTempFileName();
+            try
+            {
+                string fileName = UploadFromString(ContentsText);
+                _fileSystem.CopyToLocal(PathToFile(fileName), tempFilePath);
+                Assert.True(File.Exists(tempFilePath));
+                Assert.Equal(ContentsText, File.ReadAllText(tempFilePath));
+            }
+            finally
+            {
+                File.Delete(tempFilePath);
+            }
+        }
+
+        [Fact(Skip = SkipMessage)]
+        public void TestCopyFromLocalE2E()
+        {
+            var tempFilePath = Path.GetTempFileName();
+            var tempFileName = Path.GetFileName(tempFilePath);
+            try
+            {
+                File.WriteAllText(tempFilePath, ContentsText);
+                Uri remoteFileUri = PathToFile($"/{defaultFolderName}/{tempFileName}");
+                _fileSystem.CopyFromLocal(tempFilePath, remoteFileUri);
+                Assert.True(_adlsClient.CheckExists($"/{defaultFolderName}/{tempFileName}"));
+                var stream = _fileSystem.Open(remoteFileUri);
+                StreamReader reader = new StreamReader(stream);
```

Comment:
nit: using all these temporary variables decreases readability. If you're not going to use the variable I suggest just simplifying the call.
```csharp
string streamText = new StreamReader(stream).ReadToEnd();
```

Reply:
fixed